springdb/frontend/src/sections/@dashboard/app/CytoCanvas.js

```diff
@@ -0,0 +1,82 @@
+import cytoscape from 'cytoscape';
+
+import {Component} from 'react';
+
+// ----------------------------------------------------------------------
+
+export default class CytoCanvas extends Component {
+    render(){
+        
+        return (
+                <div id="cyto_canvas" style={this.canvasStyle} ref={el => this.el = el} /> 
+        );
+    }
+
+    componentDidMount() {
+        let cytoInstance = cytoscape({
+            elements: this.props.graphNodes,
+            container: this.el,
+            style: this.initialStyles,
+
+            layout: {
+                name: 'circle',
+            }
+        })
+        this.setState({
+            cytoInstance: cytoInstance
+        })
+    }
+
+    componentWillUnmount(){
+        this.state.cytoInstance.unmount();
```

Comment:
Seems like we might want destory() here since we won't be re-using the instance?